<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseDialog</class>
 <widget class="QWidget" name="DatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navdatabase Loader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="labelDatabaseInformation">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditDatabaseSceneryFile"/>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelDatabaseBasePath">
       <property name="text">
        <string>&amp;Flight Simulator Base Path:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditDatabaseBasePath</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelDatabaseSceneryFile">
       <property name="text">
        <string>Scenery &amp;Configuration File:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditDatabaseSceneryFile</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditDatabaseBasePath"/>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QPushButton" name="pushButtonDatabaseSceneryFile">
       <property name="text">
        <string>Select ...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPushButton" name="pushButtonDatabaseBasePath">
       <property name="text">
        <string>Select ...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxSimulator"/>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QPushButton" name="pushButtonDatabaseResetPaths">
       <property name="text">
        <string>&amp;Reset Paths</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelDatabaseSimulator">
       <property name="text">
        <string>&amp;Simulator:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxSimulator</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxDatabase">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxSimulator</tabstop>
  <tabstop>pushButtonDatabaseResetPaths</tabstop>
  <tabstop>lineEditDatabaseBasePath</tabstop>
  <tabstop>pushButtonDatabaseBasePath</tabstop>
  <tabstop>lineEditDatabaseSceneryFile</tabstop>
  <tabstop>pushButtonDatabaseSceneryFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
